src/util.ts

```diff
@@ -1477,7 +1489,7 @@ export class Util {
 
     public validateTooDeepFile(file: (BrsFile | XmlFile)) {
         //find any files nested too deep
-        let pkgPath = file.pkgPath ?? file.pkgPath.toString();
+        let pkgPath = file.pkgPath ?? (file.pkgPath as any).toString();
```

Comment:
The type of `file.pkgPath` is `string`, so unless something unexpected has happened this `toString()` call isn't necessary. Since it seems to only be relevant to cases where the types are wrong I don't think that an `any` cast will hurt anything.